packages/wallets/src/features/cashier/modules/WithdrawalFiat/WithdrawalFiat.tsx

```diff
@@ -30,6 +29,7 @@ const WithdrawalFiat: React.FC<WithdrawalFiatProps> = ({ verificationCode }) =>
             {iframeUrl && (
                 <iframe
                     className='wallets-withdrawal-fiat__iframe'
+                    data-testid='dt_wallets-withdrawal-fiat-iframe'
```

Comment:
```suggestion
                    data-testid='dt_wallets_withdrawal_fiat_iframe'
```
https://deriv-group.slack.com/archives/C3AHMSM3P/p1702363978866289